pkg/hub/controllers/cluster/reconciler.go

```diff
@@ -50,14 +50,20 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 	if cr.Status.ClusterHealth == nil || time.Since(cr.Status.ClusterHealth.LastUpdated.Time) > 3*time.Minute {
 		log.Info("updating cluster info on controller")
 		if err := r.updateClusterInfo(ctx, cr); err != nil {
+			// Skip the error and continue health check
```

Comment:
Why we are not reconciling if `err != nil` so if update fails due to conflicts or some other reason the cluster info will not populate in the cluster CR and in the UI it will not show any data this could be a bad user experience because without that user could not able to register the cluster. Since it needs nodeIP in the cluster CR spec.